app/src/main/java/com/internship/move/SplashFragment.kt

```diff
@@ -0,0 +1,22 @@
+package com.internship.move
+
+import android.os.Bundle
+import android.os.Handler
+import android.os.Looper
+import android.view.View
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+
+class SplashFragment : Fragment(R.layout.fragment_splash) {
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        Handler(Looper.getMainLooper()).postDelayed({
```

Comment:
Leaving a line between the super call and the rest of the method code can make it easier to read
(this is subjective though, and wouldn't apply it when you actually have to change the super call)

Reply:
I forgot to add it, my fault here :)

Reply:
No worries, it's just guidelines that become more automated in your coding style with time :P